test/parallel/sdk.API.getInfo.test.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ *
+ * @group sdk
+ * @group parallel
+ */
+import { jest } from "@jest/globals";
+import { Keyring } from "@polkadot/api";
+import { getApi, getMangataInstance, initApi } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { BN } from "@polkadot/util";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { User } from "../../utils/User";
+import { getEnvironmentRequiredVars, stringToBN } from "../../utils/utils";
+import { Xyk } from "../../utils/xyk";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { getLiquidityAssetId } from "../../utils/tx";
+import {
+  BN_BILLION,
+  BN_ZERO,
+  MangataInstance,
+  PoolWithRatio,
+} from "@mangata-finance/sdk";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(2500000);
+process.env.NODE_ENV = "test";
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+let testUser: User;
+let sudo: User;
+let keyring: Keyring;
+let token1: BN;
+let liqId: BN;
+let mangata: MangataInstance;
+const defaultCurrencyValue = new BN(250000);
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+  keyring = new Keyring({ type: "sr25519" });
+
+  // setup users
+  sudo = new User(keyring, sudoUserName);
+
+  [testUser] = setupUsers();
+
+  await setupApi();
+
+  [token1] = await Assets.setupUserWithCurrencies(
+    sudo,
+    [defaultCurrencyValue],
+    sudo
+  );
+  mangata = await getMangataInstance();
+  await Sudo.batchAsSudoFinalized(
+    Assets.FinalizeTge(),
+    Assets.initIssuance(),
+    Assets.mintNative(testUser),
+    Assets.mintToken(token1, testUser, Assets.DEFAULT_AMOUNT),
+    Sudo.sudoAs(
+      testUser,
+      Xyk.createPool(
+        MGA_ASSET_ID,
+        Assets.DEFAULT_AMOUNT.divn(2),
+        token1,
+        Assets.DEFAULT_AMOUNT.divn(2)
+      )
+    )
+  );
+
+  liqId = await getLiquidityAssetId(MGA_ASSET_ID, token1);
+
+  await Sudo.batchAsSudoFinalized(Assets.promotePool(liqId.toNumber(), 20));
+});
+
+test("getAmountOfTokensInPool return poolAmount", async () => {
```

Comment:
can you create the test for the reversed tokens?  this one is about, (MGA, token1) , add a test for ( token1, MGA ) plz. behavior should be the same AFIK

Reply:
Added